craft_parts/plugins/autotools_plugin.py

```diff
@@ -31,6 +31,7 @@ class AutotoolsPluginProperties(PluginProperties, frozen=True):
 
     autotools_configure_parameters: list[str] = []
     autotools_bootstrap_parameters: list[str] = []
+    autotools_parallel: bool = True
```

Comment:
```suggestion
    disable_parallel: bool = False
```

Shouldn't this just be [the standard `disable-parallel`](https://canonical-craft-parts.readthedocs-hosted.com/latest/common/craft-parts/reference/part_properties/#disable-parallel) parameter?